core/commands.go

```diff
@@ -469,6 +469,15 @@ var (
 		Info: "PERSIST removes the expiration from a key",
 		Eval: evalPersist,
 	}
+	expirateCmdMeta = DiceCmdMeta{
+		Name:  "EXPIREAT",
+		Info:  `EXPIREAT sets the expiry time of the specified key, in Unix timestamp format.`,
+		Eval:  evalEXPIREAT,
+		Arity: 2,
+		KeySpecs: KeySpecs{
+			BeginIndex: 0, // 过期时间的参数从第一个参数开始
```

Comment:
I would recommend using English for code comments so that they are accessible to the rest of the community :)